schema.sql

```diff
@@ -10,4 +10,33 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals
-    ADD COLUMN species VARCHAR(100);
\ No newline at end of file
+    ADD COLUMN species VARCHAR(100);
+
+CREATE TABLE owners(
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(100) NOT NULL,
+    age INT 
+);
```

Comment:
[OPTIONAL] You have done a great job on this 👍 Well done 🔥. I wanted to draw your attention to using ```IDENTITY``` instead of ```SERIAL```.  While ```SERIAL``` works perfectly, ```IDENTITY``` gives you more control over the auto-increment value and allows you to control what scale you want your sequence to be increased by. You can add this to the commands you have in your arsenal 😉. Also, the ```IDENTITY``` columns are compliant with SQL standards. I will heavily encourage you to read [this article](https://wanago.io/2022/02/21/serial-type-identity-columns-postgresql-typeorm/#:~:text=As%20opposed%20to%20the%20serial,compliant%20with%20the%20SQL%20standard.&text=We%20can%20provide%20different%20properties,by%202%20instead%20of%201.) which personally helped me greatly to understand this concept